server/sockets/index.js

```diff
@@ -69,38 +70,57 @@ const socketListeners = socketInstance => {
     await associateSocketWithSession(socket);
     await joinMetaDataRooms(socket);
 
-    socket.on('joinRoom', ({ data, room }) => {
-      const { roomId, userId, viewId } = data;
-
-      switch (room) {
-        case Routes.LIST:
-          socket.join(roomId);
-          listViewClients.set(userId, { socketId: socket.id, viewId });
-          break;
-        case Routes.COHORT:
-          socket.join(roomId);
-          cohortViewClients.set(userId, { socketId: socket.id, viewId });
-          break;
-        default:
-          break;
+    socket.on(AppEvents.JOIN_ROOM, emittedData => {
+      // This check is for the old functionality to work.
+      // After refactoring this handler will be simplified to:
+      // socket.on('joinRoom', room => socket.join(room));
```

Comment:
when adding block comments user block comments. Just type in VSCode   -> `/**` and hit enter
```

/**
 * This check is for the old functionality to work.
 * After refactoring this handler will be simplified to:
 * switch (room) {	      // socket.on('joinRoom', room => socket.join(room));
 */
```

and please add sentence  " Switch statement will be removed"